src/WebManager/Controllers/ApiController.cs

```diff
@@ -78,6 +80,33 @@ public RegistryCredential GetDockerCredential()
             };
         }
 
+    
+
+        [HttpPut]
+        public async Task<IActionResult> Manifest(string repo, string tag,  [FromBody] string manifest)
+        {
+
+            string json = manifest;
```

Comment:
nit: we usually prefer using `var` instead of the given type, as you did a few lines below this one; furthermore, it's not clear what the purpose of this variable is since it's simply referencing `manifest`. Could you have simply used `manifest` below instead of `json`?